docs/developer/stock-allocation.mdx

```diff
@@ -43,13 +45,32 @@ classDiagram
     class Warehouse{
       click_and_collect_option: String
     }
+    class Channel{
+      allocationStrategy: String
+    }
 ```
 
+### Allocation strategies
```

Comment:
Perhaps this should be nested under the `Allocation flow` since the later is broader and provides more context?

Reply:
Moved